### ISSUE TITLE
Escaping of single quotes in file names

### ISSUE THREAD

**Issue by satta** [nextflow-io/nextflow#99]
I noticed that my pipeline behaved strangely when filenames with single quotes (containing the strings `3'` and `5'`) were received in a process using `file '*.foo' from channel`. Turns out to stage these in the process work directory, the following is done by Nextflow (example):

```
$ cat .command.run
...
ln -s '/Users/ss34/work/9d/080b40902484930e9b28181b6b1e0e/test'file.out' '1.out'
ln -s '/Users/ss34/work/9d/080b40902484930e9b28181b6b1e0e/testfile2.out' '2.out'
...
```
in which the first line looks suspicious. Maybe the input file names should be escaped before trying to symlink them in this fashion.

Here's a test case to reproduce the issue: https://gist.github.com/satta/0726c1f5088c713d320a
The pipeline finishes fine after I remove the single quote in the first filename.

**Comment by pditommaso** [comment]
Ouch! You are right, file names should be escaped.

**Comment by pditommaso** [comment]
Fixed in version `0.17.0`
